examples/hana-scale-up-perf-optimized-azure.yaml

```diff
@@ -450,6 +450,32 @@ groups:
           ## Remediation
           Update SAP HANA to a supported version.
           
+          ## References
+          - section 2 in https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
+        scored: true
+      - id: 2.2.10
+        description: "HANA's autostart is disabled"
+        audit: |
+          sid=$(sudo crm configure show | grep -m1 SID= | sed -e "s/.*SID=\(...\).*/\1/" | tr '[:lower:]' '[:upper:]')
+          inst=$(sudo crm configure show | grep -m1 InstanceNumber= | sed -e "s/.*InstanceNumber=\(..\).*/\1/")
+          cat /hana/shared/${sid}/profile/${sid}_HDB${inst}_$(hostname)
```

Comment:
I'm not sure if using `/hana/shared` is a good idea. I think this folder is configurable. We could use `/usr/sap/${sid}/SYS/profile` instead, which is always the same

Reply:
Thank you! This makes sense. Changed.